package.json

```diff
@@ -94,31 +96,42 @@
     "@types/d3-shape": "^2.1.0",
     "@types/d3-zoom": "^1.7.5",
     "@types/highlight.js": "^10.1.0",
-    "@types/jest": "^27.4.1",
+    "@types/jest": "^28.0.0",
     "@types/lodash-es": "^4.17.7",
     "@types/mixpanel-browser": "^2.38.1",
     "@types/node": "^18.15.11",
     "@types/uuid": "^8.3.4",
     "@types/webpack-env": "^1.18.0",
-    "codelyzer": "^6.0.2",
+    "@typescript-eslint/eslint-plugin": "5.43.0",
+    "@typescript-eslint/parser": "5.43.0",
     "commitizen": "^4.3.0",
     "cz-conventional-changelog": "^3.3.0",
+    "eslint": "^8.28.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "2.27.5",
+    "eslint-plugin-jsdoc": "40.0.0",
+    "eslint-plugin-no-null": "1.0.2",
```

Comment:
Why do we install this, but then remove all the comments from tslint that enable this behavior?
```typescript
// tslint:disable-next-line: no-null-keyword
```
I think we don't actually need to install this plugin if we removed all the comments.

Also, FWIW, I think allowing the `null` keyword is a great thing.